workspaces/default/themes/base/partials/header.html

```diff
@@ -9,15 +9,15 @@ <h1 class="brand"><a href=""><img class="logo" src="{{theme.images.logo}}" alt="
       </div>
 
       <input type="checkbox" id="header_nav" class="menu-toggle-checkbox" />
-      <nav class="eight columns" role="navigation">
-
+      <nav class="eight columns" role="navigation" aria-labelledby="top-navigation">
+        <div id="top-navigation" hidden>Header Navigation</div>
```

Comment:
Do we have to do `aria-labelledby` here with a hidden element or can we just do `aria-label="Header Navigation"`?